### ISSUE TITLE
Extensible streamed media info in M3U playlist

### ISSUE THREAD

**Issue by octavian-paraschiv** [octavian-paraschiv/protone-suite#288]
Add the possibility to store extensible media info in M3U playlist.
Direct applicability is for streamed media such as Shoutcast radio stations, Deezer media and Galaxy media.

Media on local disks remains as spec'ed in https://en.wikipedia.org/wiki/M3U:
```
#EXTINF:(duration_in_sec),(title)
(path)
```
Galaxy media:
```
#GALAXY:{json_data}
(galaxy_media_url)
```
Deezer media - new format:
```
#DZMEDIA:JSON:{json_data}
(dzmedia_url)
```
Deezer media (legacy playlists saved with earlier ProTONE versions):
```
#DZMEDIA:(duration_in_sec),(title),(other_info_as_sring_array)
(dzmedia_url)
```
Shoutcast radio station - new format:
```
#RADIO:JSON:{json_data}
(radio_station_url)
```
Shoutcast radio station (legacy playlists saved with earlier ProTONE versions):
```
#RADIO:0,(description)
(radio_station_url)
```

**Comment by octavian-paraschiv** [comment]
This feature is available in ProTONE builds >= 3.1.75. Needs confirmation with a release version.

**Comment by octavian-paraschiv** [comment]
verified with build 3.1.100
